src/Mod/AddonManager/AddonManager.py

```diff
@@ -72,8 +72,9 @@ def GetResources(self):
     def Activated(self):
 
         # display first use dialog if needed
-        readWarning = FreeCAD.ParamGet("User parameter:Plugins/addonsRepository").GetBool("readWarning",
-                                                                                          False)
+        readWarningParameter = FreeCAD.ParamGet("User parameter:BaseApp/Preferences/Addons")
+        readWarning = readWarningParameter.GetBool("readWarning", False)
+        readWarning |= FreeCAD.ParamGet('User parameter:Plugins/addonsRepository', False)
```

Comment:
Sorry, I didn't look at this closely enough before. Line 77 should really be two lines, one to fetch the new storage location, and a second line to access the specific stored variable (I haven't tested this yet, though):
```suggestion
        newReadWarningParameter = FreeCAD.ParamGet("User parameter:Plugins/addonsRepository")
        readWarning |= newReadWarningParameter.GetBool("readWarning", False)
```